Yoast/wordpress-seo
# Change CRLF to LF line endings in the issue template files

## Summary

This PR can be summarized in the following changelog entry:

* Change CRLF to LF line endings in the issue template files

## Test instructions

Verify line endings are changed from CRLF to LF (your editor / IDE should show this somewhere in the UI).

Note: I guess when these files are modified via the web interface, they get CRLF line endings. As a consequence, every time I switch to a branch, Git warns me about "modified files" e.g.:
`modified:   .github/ISSUE_TEMPLATE/Bug_report.md`

This may also depend on global / local GIt settings so not everyone will get the same warning but s a best practice it would be nice to keep line endings consistent across the project files.
